lib/dataLayer.tsx

```diff
@@ -421,53 +475,99 @@ function _buildFormDataObject(form: PublicFormSchemaProperties, values: Response
   const formData = new FormData();
   form.elements = form.elements.filter((element) => !["richText"].includes(element.type));
```

Comment:
We need to be careful when we modify a passed in object to a function.  This does modify the object higher up in the scope.  In this case it flows all the way back up to `formikBag.props`.  If we're not sure if modifying the object higher in the scope will cause issues it's aways better to clone it and then do manipulations.
In the scope of this function it seems like we're only using `form.elements` so we could just create a const for the manipualted array and leverage it in the scope wihtin this function.